sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -163,6 +163,49 @@ namespace Azure { namespace Storage {
         std::string LeaseId;
       };
 
+      /**
+       * @brief An Azure Storage blob.
+       */
+      struct BlobItem final
+      {
+        /**
+         * Blob name.
+         */
+        std::string Name;
+        /**
+         * Indicates whether this blob was deleted.
+         */
+        bool IsDeleted = bool();
```

Comment:
Instead of using the function call form of the constructor, use the default value initializer.
```suggestion
        bool IsDeleted{};
```

Reply:
It doesn't make any difference for primitive types, does it?

Reply:
@ahsonkhan Do you have a preference on this one?

Most of the use cases are in generated code, I'd like to keep consistent. If we were to fix it, I'd like to fix them all. We'll need to update CodeGen first.

Reply:
There are C++ core guidelines that strongly recommend using initializer syntax (`{}`) over assignment initialization (`=xxx`) or function initialization (`()`). 

That's because assignment initialization is easily confused with assignment, and function initialization is easily confused with a function call. Initializer syntax is specific and unique.

However, if this is generated code, it's up to the generator